userguide/intro.rst

```diff
@@ -414,6 +414,8 @@ U2
   :menuselection:`Dashboard`
   now change color to reflect the current pool status.
 
+* Now allows the rollback of any snapshot.
```

Comment:
This confuses me: What allows the rollback, where is the change located in the UI, rollback of the snapshot or freenas itself?